Mage.Sets/src/mage/cards/t/TergridGodOfFright.java

```diff
@@ -172,4 +180,111 @@ public boolean apply(Game game, Ability source) {
         }
         return false;
     }
-}
\ No newline at end of file
+}
+
+// Based on SacrificeTargetCost
```

Comment:
Do not copy-paste big effects/abilities for one line change. If it wrongly works in some use cases then try to improve it first, e.g. by additional param or use case processing.